### PR TITLE
Add PBR parameters

### LINKED ISSUE #742: PBR lacks parameters
Weapon material artists be like: hey I wanna be able to change emission with proxies.

I propose the color parameters MRAO Scale and Emission Scale should be added to the PBR shader.

In usual fashion, I've already made these: https://github.com/momentum-mod/game/tree/feature/pbr-params

### PR DESCRIPTION
**Pull request by bonjorno7**
Closes #742 

I added MRAO Scale and Emission Scale to the PS30 version of the PBR shader.
No love for PS20b, just doesn't have enough registers for it.

### Checklist
- [x] If there is a localization token change, I have updated the `momentum_english_ref.res` file with the changes, ran `tokenizer.py` to generate an up-to-date localization file, and have committed both the `.res` file changes and the new localization `.txt` file
- [x] If I introduced new h/cpp files, I have added them to the appropriate project's VPC file (`server_momentum.vpc` / `client_momentum.vpc` / etc)
- [x] My commits are relatively small and scoped to the best of my ability
- [x] My branch has a clear history of changes that can be easy to follow when being reviewed commit-by-commit
- [x] My branch is functionally complete; the only changes to be done will be those potentially requested in code review

### CODE REVIEW

File: mp/src/materialsystem/stdshaders/pbr_dx9.cpp [hunk 1]
```diff
@@ -359,6 +367,33 @@ BEGIN_VS_SHADER(PBR, "PBR shader")
...
+            {
+                // Setting up mrao scale
+                Vector mraoScale;
+                if (bHasMraoScale)
+                {
+                    params[info.mraoScale]->GetVecValue(mraoScale.Base(), 3);
+                }
+                else
+                {
+                    color = Vector{ 1.f, 1.f, 1.f };
```
Comment (Gocnak) [inline mp/src/materialsystem/stdshaders/pbr_dx9.cpp:380]: Should this be
```suggestion
                    mraoScale = Vector{ 1.f, 1.f, 1.f };
```
?

File: mp/src/materialsystem/stdshaders/pbr_dx9.cpp [hunk 2]
```diff
@@ -359,6 +367,33 @@ BEGIN_VS_SHADER(PBR, "PBR shader")
...
+
+                // Setting up emission scale
+                Vector emissionScale;
+                if (bHasEmissionScale)
+                {
+                    params[info.emissionScale]->GetVecValue(emissionScale.Base(), 3);
+                }
+                else
+                {
+                    color = Vector{ 1.f, 1.f, 1.f };
```
Comment (Gocnak) [inline mp/src/materialsystem/stdshaders/pbr_dx9.cpp:392]: Should this be
```suggestion
                    emissionScale = Vector{ 1.f, 1.f, 1.f };
```
?

File: mp/src/materialsystem/stdshaders/pbr_dx9.cpp (hunk 1 above)
Comment (bonjorno7) [reply mp/src/materialsystem/stdshaders/pbr_dx9.cpp:380]: Good catch

File: mp/src/materialsystem/stdshaders/pbr_dx9.cpp (hunk 2 above)
Comment (bonjorno7) [reply mp/src/materialsystem/stdshaders/pbr_dx9.cpp:392]: I'll fix this first thing tomorrow if you haven't already by then

Comment (bonjorno7) [reply mp/src/materialsystem/stdshaders/pbr_dx9.cpp]: Done in force push

File: mp/src/materialsystem/stdshaders/pbr_dx9.cpp (hunk 1 above)
Comment (bonjorno7) [reply mp/src/materialsystem/stdshaders/pbr_dx9.cpp]: Done in force push

**Comment by bonjorno7** [comment]
Btw I've found that the else part of those if statements is literally never reached no matter what you do, but I'm leaving them anyway just in case.

**Comment by Gocnak** [comment]
> 
> 
> Btw I've found that the else part of those if statements is literally never reached no matter what you do, but I'm leaving them anyway just in case.

Makes sense, you gave the shader params default values, I assume it uses those when they aren't actually in the vmt

**Pull request merged**